.ansible-pulp_tox.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# wrapper for inserting pulp_rpm_prerequisites into the ansible-pulp CI
+cd ..
+if [ ! -e ansible-pulp ]; then
+  git clone https://github.com/pulp/ansible-pulp
+fi
+cd ansible-pulp
+if [ ! -e roles/pulp.pulp_rpm_prerequisites ]; then
+  ln -s $TRAVIS_BUILD_DIR roles/pulp.pulp_rpm_prerequisites
+fi
+
+find ./ -type f -exec sed -i 's/pulp_use_system_wide_pkgs:\sfalse/pulp_use_system_wide_pkgs: true/g' {} \;
+find ./ -type f -exec sed -i 's/pulp_install_plugins:\s{\}/pulp_install_plugins:\n  pulp-rpm:\n    prereq_role: "pulp.pulp_rpm_prerequisites"/g' {} \;
```

Comment:
Instead of modifying the default variables in the roles, use yq to add/remove/modify values to the group_vars files:
./example-source/group_vars/all
./example-use/group_vars/all

It may be possible to add it to this instead (in case it has higher precedence), that would be better, but not required:
./molecule/scenario_resources/group_vars

If neither of those work, you can just use the find and sed on those files.